CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
+- add support for CLI calls. ![\25](https://github.com/mllam/mllam-data-prep/pull/25).
```

Comment:
maybe it would be clearer to say "add support for calling cli from python function and add tests for cli with/without `dask.distributed`

Reply:
Yeah, I updated it. Please check.

Reply:
maybe we should just write what the CLI call is now? Is it still `python -m ...`?

Maybe the changelog should then say 

> add new CLI call option `...` (so that `python -m mllam_data_prep ...` isn't needed)

or something like that?